helm/defectdojo/templates/secret-postgresql.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: defectdojo-postgresql-specific
+  name: {{ .Values.postgresql.existingSecret }}
```

Comment:
Is "existingSecret" not a bit misleading? If I understood it correct the secret is created always, but in one case the password is given and not generated.

Reply:
That's right, the previous implementation was expecting to have the secret created upfront, outside of the helm chart, which is not quite self-contained for the case when you want to install it for instance with helmfile in one go, and it uses an external database.

Reply:
Should we rename the variable to "secret" ?

Reply:
I can rename them.

Reply:
It should be renamed now.